src/services/interfaces/ICrossMarginService.sol

```diff
@@ -23,6 +23,16 @@ interface ICrossMarginService {
     uint256 minToAmount;
   }
 
+
+  struct TransferCollateralParams {
+    address fromPrimaryAccount;
+    uint8 fromSubAccountId;
+    address toPrimaryAccount;
```

Comment:
do we allow to transfer between account ?

Reply:
yes, for future support